src/app/pages/AccountPage/index.tsx

```diff
@@ -79,11 +79,12 @@ export function AccountPage(props: Props) {
   const selectedNetwork = useSelector(selectSelectedNetwork)
   const { active } = useSelector(selectTransaction)
   const wallets = useSelector(selectWallets)
+  const activeWallet = useSelector(selectActiveWallet)
 
   const balanceDelegations = stake.delegations.reduce((acc, v) => acc + Number(v.amount), 0)
   const balanceDebondingDelegations = stake.debondingDelegations.reduce((acc, v) => acc + Number(v.amount), 0)
   const balance: BalanceDetails = {
-    available: account.liquid_balance ?? 0,
+    available: parseFloat(activeWallet?.balance.available ?? '0'), // Use local oasis-node available balance.
```

Comment:
This would need to check `activeWallet?.address === account.address`
- If user searches for address without opening a wallet: activeWallet is undefined
- If user opens a wallet and searches for different address, activeWallet is user's wallet